lib/infra/infra-stack.ts

```diff
@@ -159,13 +160,21 @@ export class InfraStack extends Stack {
     });
 
     if (!props.securityDisabled && !props.minDistribution) {
-      opensearchListener = nlb.addListener('elasticsearch', {
+      opensearchListener = nlb.addListener('elasticsearchHTTPS', {
         port: 443,
         protocol: Protocol.TCP,
       });
+      opensearchListener19200 = nlb.addListener('elasticsearchHTTP', {
+        port: 80, // or some other port that makes sense in this context
+        protocol: Protocol.TCP,
+      });
```

Comment:
This seems a bit odd to me, to just assign one to the http port and another to the https port. Since we have to specify the port for the load balancer endpoint now regardless, we should probably just remove this if else structure and just use lines 172-179

Reply:
That makes sense! Done! Thanks Tanner!